src/pages/docs_sdk_forgot-password.html

```diff
@@ -0,0 +1,44 @@
+<div class="section">
+  <h2><a href="/docs/">Docs</a> : <a href="/docs/sdk/">SDK</a> : forgotPassword</h2>
+
+  <p><span class="font-semibold">forgotPassword</span> lets you send a temporary password to the user's email address. This API will return a promise that gets resolved once the email with the temporary password has been sent.</p>
+
+  <pre>
+    <code class="language-javascript">userbase.forgotPassword({ username })
+      .then((user) => {
+        // email with temporary password sent
+      })
+      .catch((e) => console.error(e))</code>
+  </pre>
+
+  <h3 id="params">Parameters</h3>
+
+  <ul>
+    <li>
+      <span class="field">username</span> [string | Len: 1-50] - The username for the account to send a temporary password to.
```

Comment:
Need to change this elsewhere too, but the max length we enforce server-side is actually 100